src/styles/main.scss

```diff
@@ -1,3 +1,31 @@
+@import url("./utils/_variables.scss");
+@import url("./blocks/header.scss");
+@import url("./blocks/card.scss");
+
+* {
+  margin: 0;
+}
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  min-height: 100vh;
+}
+
+a {
+  text-decoration: none;
+  color: var(--main-accent);
+}
+
+li {
+  list-style: none;
+}
+
+.main {
+  &__container {
+    max-width: 944px;
+    display: flex;
+    justify-content: center;
+    margin: auto;
+  }
 }
```

Comment:
yeah, that's a great point from @SalehBUD 👍

Reply:
> yeah, that's a great point from @SalehBUD 👍

I need more information on this.

I think that just to put the code that is repeated "in a separated file called _extends inside utils" without using a Sass rule is a little vague.

Should I use @extend -  [https://sass-lang.com/documentation/at-rules/extend](url), or should @mixins - [https://sass-lang.com/documentation/at-rules/mixin](url)? Or something else?

Thank you.

Reply:
@RomeoKs this was more about general file structure - so that file `main.scss` only contains imports and nothing else
so just moving your general styles into separate file and then import into `main.scss`